src/bot.ts

```diff
@@ -75,7 +79,7 @@ async function _helpCommand (msg: Message) {
     `${fromUser ? 'Hi ' + fromUser.first_name : 'Hi there'}!
     
 I don't talk much for now. I just notify every new order in dFusion channel.
-Please, go to t.me/dFusionPoC to get notified on every new order.
+Please, go to ${channelHandle} to get notified on every new order.
```

Comment:
In case of a private chat, do we still want to display this line?

Reply:
I thought about removing it, but then, I though it's fineee for local development

Reply:
I know it doesn't make a lot of sense in terms of usability, but I try to keep it as simple as posible